admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -636,12 +636,20 @@ def _loadOneFile(self, conn, database, table, file, csvPrefix):
         """Load data from a single file into existing table"""
         cursor = conn.cursor()
 
-        # need to know field separator, default is the same as in partitioner.
-        separator = self.partOptions.get(csvPrefix + '.delimiter', '\t')
+        # need to know field delimiter, default is the same as in partitioner
+        special_chars = [self.partOptions.get(csvPrefix + '.delimiter', '\t'),
+                         self.partOptions.get(csvPrefix + '.enclose', '"'),
+                         self.partOptions.get(csvPrefix + '.escape', '\\'),
+                         self.partOptions.get(csvPrefix + '.newline', '\n')]
+
+        special_chars = map(repr, special_chars)
```

Comment:
You also need to escape single quote character `'`, and `repr` will not do that.